src/core/server/http/http_service.ts

```diff
@@ -104,5 +125,9 @@ export class HttpService implements CoreService<HttpServiceSetup, HttpServiceSta
 
     await this.httpServer.stop();
     await this.httpsRedirectServer.stop();
+    for (const [port, server] of this.secondaryServers.entries()) {
```

Comment:
Using `await` within `for..of` executes this "serially". Could we instead using `Promise.all` to run "in parallel"?

await Promise.all([...this.secondaryServers.entries()].map(async ([port, server]) => {
  await server.stop();
  this.secondaryServers.delete(port);
}));

Reply:
Should I break this up or are we ok with this nested array generation? 

`await Promise.all([...this.secondaryServers.values()].map(s => s.stop()));`